src/Avalonia.Controls/Window.cs

```diff
@@ -183,6 +183,15 @@ public WindowIcon Icon
             set { SetValue(IconProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the startup location of the window.
+        /// </summary>
+        public WindowStartupLocation WindowStartupLocation
```

Comment:
I think this needs to be an `AvaloniaProperty` so that it can be bound (I know it doesn't sound likely that someone will want to bind this one, but someone will ;) )